ui-admin/src/HomePage.tsx

```diff
@@ -28,8 +28,10 @@ function HomePage() {
     <div className="ms-5 mt-4">
       <h2 className="h4">My Studies</h2>
       <ul className="list-group list-group-flush fs-5">
-        { portalList.sort((a, b) => a.name.localeCompare(b.name)).flatMap(portal =>
-          portal.portalStudies.sort((a, b) => a.study.name.localeCompare(b.study.name)).map(portalStudy => {
+        { portalList.sort((a, b) => a.name.localeCompare(b.name)).flatMap(portal => [
```

Comment:
I think this would be semantically clearer and more accessible via screenreader if you treated this as a list-of-lists, rather than putting titles in interstitial area
```suggestion
        { portalList.sort((a, b) => a.name.localeCompare(b.name)).flatMap(portal => 
          <li key = "portal.shortcode">
          <span className={'mt-2'}>{portal.name}</span>
            <ul className="list-group list-group-flush">
              ...portal.portalStudies.sort((a, b) =>
```